quiz/migrations/0004_contestant_complete.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.3 on 2017-10-22 15:54
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('quiz', '0003_auto_20170504_1409'),
```

Comment:
Is that string the DB ID or something similar? Not a code issue, just wondering.

Reply:
Django migrations propagate changes to models to the db.
They are automatically generated, including this string which references the previous migration.
Here is some documentation on [migrations](https://docs.djangoproject.com/en/1.11/topics/migrations/).
As long as we trust the django migration tool then this pull request should be fine! :)

Reply:
Cool, thanks for explaining!